<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=robots content="all,follow"><meta name=googlebot content="index,follow,snippet,archive"><meta name=viewport content="width=device-width,initial-scale=1"><title>TKO_3121: Topics</title>
<meta name=author content="DevCows"><meta name=keywords content="devows,hugo,go"><meta name=description content="Site template made by devcows using hugo"><meta name=generator content="Hugo 0.133.1"><link href='//fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,500,700,800' rel=stylesheet type=text/css><link rel=stylesheet href=//use.fontawesome.com/releases/v5.11.2/css/all.css><link rel=stylesheet href=//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css integrity=sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u crossorigin=anonymous><link href=/css/animate.css rel=stylesheet><link href=/css/style.default.css rel=stylesheet id=theme-stylesheet><link href=/css/custom.css rel=stylesheet><!--[if lt IE 9]><script src=https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js></script><script src=https://oss.maxcdn.com/respond/1.4.2/respond.min.js></script><![endif]--><link rel="shortcut icon" href=/img/favicon.ico type=image/x-icon><link rel=apple-touch-icon href=/img/apple-touch-icon.png><link href=/css/owl.carousel.css rel=stylesheet><link href=/css/owl.theme.css rel=stylesheet><link rel=alternate href=http://datascience.utu.fi/index.xml type=application/rss+xml title="Turku Data Science Group"><meta name=twitter:card content="summary"><meta name=twitter:site content="@GoHugoIO"><meta name=twitter:title content="TKO_3121: Topics"><meta name=twitter:description content="Site template made by devcows using hugo"><script async src="https://www.googletagmanager.com/gtag/js?id=G-3FJ60G61DS"></script><script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments)}gtag("js",new Date),gtag("config","G-3FJ60G61DS"),console.log("Test Google Analytics")</script><script type=text/javascript>_mtm=window._mtm=window._mtm||[],_mtm.push({"mtm.startTime":(new Date).getTime(),event:"mtm.Start"});var _mtm,d=document,g=d.createElement("script"),s=d.getElementsByTagName("script")[0];g.type="text/javascript",g.async=!0,g.src="https://analytics.utu.fi/js/container_5AYuB7oa.js",s.parentNode.insertBefore(g,s)</script></head><body><div id=all><header class=navbar-affixed-top data-spy=affix data-offset-top=62><div class="navbar navbar-default yamm" role=navigation style=padding-left:0 id=navbar><div class=container><div class=navbar-header><a class="navbar-brand home" href=/><img src=/img/logopic/university_turku.png alt="TKO_3121: Topics logo" class="hidden-xs hidden-sm university-logo">
<span class=sr-only>TKO_3121: Topics - go to homepage</span></a><div class=navbar-buttons><button type=button class="navbar-toggle btn-template-main" data-toggle=collapse data-target=#navigation>
<span class=sr-only>Toggle Navigation</span>
<i class="fas fa-align-justify"></i></button></div><a href=/><div class=title>Turku Data Science Group</div></a></div><div class="navbar-collapse collapse" id=navigation><ul class="nav navbar-nav navbar-right"><li class=dropdown><li><a href=/>Home</a></li></li><li class=dropdown><li class=dropdown-list><a class=dropbtn href=/research-topics>Research</a></li></li><li class=dropdown><li class=dropdown-list><a class=dropbtn href=/edu-topics>Edu</a></li><li class=dropdown><li><a href=/code/>Code</a></li></li><li class=dropdown><li><a href=/team/>Team</a></li></li><li class=dropdown><li><a href=/contact/>Contact</a></li></li></ul></div><div class="collapse clearfix" id=search><form class=navbar-form role=search><div class=input-group><input type=text class=form-control placeholder=Search>
<span class=input-group-btn><button type=submit class="btn btn-template-main"><i class="fas fa-search"></i></button></span></div></form></div></div></div></header><div id=heading-breadcrumbs><div class=container><div class=row><div class=col-md-12><h1>TKO_3121: Topics</h1></div></div></div></div><div id=content><div class=container><div class=row><div class=col-md-9 id=blog-post><div id=post-content><h1 id=machine-learning-and-algorithmics-seminar-2020>Machine Learning and Algorithmics Seminar 2020</h1><ul><li><a href=https://opas.peppi.utu.fi/fi/opintojakso/TKO_3121/3255>Course page (Peppi)</a></li><li><a href="https://moodle.utu.fi/course/view.php?id=17982">Moodle for Machine Learning and Algorithmics Seminar 2020</a></li></ul><h2 id=proposed-seminar-topics>Proposed seminar topics</h2><h3 id=session-1-introduction>Session 1: Introduction</h3><ul><li>Participants</li><li>Overview and practicalities</li><li>Assigning the topics</li></ul><h3 id=session-2-overview>Session 2: Overview</h3><ul><li>Introduction lecture: contemporary themes in machine learning</li><li>Discussion</li><li>Finalizing the schedule</li></ul><h3 id=session-3-position-papers>Session 3: Position papers</h3><p>Replication, Communication, and the Population Dynamics of Scientific Discovery.
McElreath, R., & Smaldino, P. E. (2015).</p><p>Machine behaviour.
Rahwan, I., Cebrian, M., Obradovich, N., Bongard, J., Bonnefon, J., & Breazeal, C., et al. (2019).</p><p>Beyond subjective and objective in statistics (with discussion and rejoinder).
Journal of the Royal Statistical Society A 180, 967&ndash;1033. Andrew Gelman and Christian Hennig (2017).</p><h3 id=session-4-hypothesis-testing-and-significance>Session 4: Hypothesis testing and significance</h3><p>Abandon statistical significance.
American Statistician 73(S1):235&ndash;245. Blakeley B. McShane, David Gal, Andrew Gelman, Christian Robert, and Jennifer L. Tackett, 2019.</p><p>The ASA Statement on p-Values: Context, Process, and Purpose.
Ronald L. Wasserstein & Nicole A. Lazar. Pages 129-133
<a href=https://doi.org/10.1080/00031305.2016.1154108>https://doi.org/10.1080/00031305.2016.1154108</a></p><p>Bayesian statistics
Jorge López Puga, Martin Krzywinski & Naomi Altman
Nature Methods 12:377–378, 2015.</p><p>Jaynes, E. T., 1976. `Confidence Intervals vs Bayesian Intervals,&rsquo; in Foundations of Probability Theory, Statistical Inference, and Statistical Theories of Science, W. L. Harper and C. A. Hooker (eds.), D. Reidel, Dordrecht, p. 175;
<a href=https://bayes.wustl.edu/etj/articles/confidence.pdf>https://bayes.wustl.edu/etj/articles/confidence.pdf</a></p><h3 id=session-5-prior-information>Session 5: Prior information</h3><p>The prior can often only be understood in the context of the likelihood.
Entropy 19:555, 2017. Andrew Gelman, Daniel Simpson, and Michael Betancourt.</p><p>The experiment is just as important as the likelihood in understanding the prior: A cautionary note on robust cognitive modelling. Computational Brain and Behavior. Lauren Kennedy, Daniel Simpson, and Andrew Gelman, 2019.</p><p>Sparsity information and regularization in the horseshoe and other shrinkage priors. In Electronic Journal of Statistics, 11(2):5018-5051. Juho Piironen and Aki Vehtari (2017). arXiv:1707.01694. After the article was published, the regularized horseshoe prior has been implemented in rstanarm and brms (without conditioning on sigma).</p><h3 id=session-6-visualization>Session 6: Visualization</h3><p>UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction.
Leland McInnes, John Healy, James Melville (2018)</p><p>Probabilistic principal component analysis.
Journal of the Royal Statistical Society: Series B (Statistical Methodology), 61(3), 611–622.
Tipping, M. E., & Bishop, C. M. (1999).</p><p>Bayesian Unidimensional Scaling for visualizing uncertainty in high dimensional datasets with latent ordering of observations.
BMC Bioinformatics, August, 2017.
Lan Huong Nguyen and Susan Holmes (2017)</p><h3 id=session-7-feature-selection>Session 7: Feature selection</h3><p>XGBoost: A Scalable Tree Boosting System.
In 22nd SIGKDD Conference on Knowledge Discovery and Data Mining, 2016.
Tianqi Chen and Carlos Guestrin.
<a href=https://arxiv.org/abs/1603.02754>https://arxiv.org/abs/1603.02754</a></p><p>Projective inference in high-dimensional problems: prediction and feature selection.
Juho Piironen, Markus Paasiniemi, and Aki Vehtari (2018).
arXiv:1810.02406.</p><p>Bayesian inference for spatio-temporal spike and slab priors.
In Journal of Machine Learning Research, 18(139):1-58.
Michael Riis Andersen, Aki Vehtari, Ole Winther and Lars Kai Hansen (2017).
arXiv:1509.04752.</p><h3 id=session-8-model-selection>Session 8: Model selection</h3><p>Bayesian model selection for complex dynamic systems
Christoph Mark, Claus Metzner, Lena Lautscham, Pamela L. Strissel, Reiner Strick, Ben Fabry.
Nature Communications volume 9(1803), 2018</p><p>Aki Vehtari, Andrew Gelman and Jonah Gabry (2017).
Practical Bayesian model evaluation using leave-one-out cross-validation and WAIC.
In Statistics and Computing, 27(5):1413–1432.
doi:10.1007/s11222-016-9696-4.
arXiv:1507.04544.</p><p>Comparison of Bayesian predictive methods for model selection.
Statistics and Computing, 27(3):711-735. doi:10.1007/s11222-016-9649-y.
Juho Piironen and Aki Vehtari (2017).
arXiv:1503.08650.</p><h3 id=session-9-scalable-inference>Session 9: Scalable inference</h3><p>Automatic variational inference in Stan. Neural Information Processing Systems. Alp Kucukelbir, Rajesh Ranganath, Andrew Gelman, and David Blei. 2015</p><p>Fundamentals and recent developments in approximate Bayesian computation.
Systematic Biology. 66(1),e66-e82 .doi: 10.1093/sysbio/syw077
Lintusaari, Jarno; Gutmann, Michael U.; Dutta, Ritabrata; Kaski, Samuel, Corander, Jukka (2017).</p><p>Bayesian Computing with INLA: A Review.
Håvard Rue, Andrea Riebler, Sigrunn H. Sørbye, Janine B. Illian, Daniel P. Simpson, Finn K. Lindgren.
<a href=https://arxiv.org/abs/1604.00860>https://arxiv.org/abs/1604.00860</a></p><p>Expectation propagation as a way of life: A framework for Bayesian inference on partitioned data. Journal of Machine Learning Research. 21, 1&ndash;53.
Aki Vehtari, Andrew Gelman, Tuomas Sivula, Pasi Jylanki, Dustin Tran, Swupnil Sahai, Paul Blomstedt, John P. Cunningham, David Schiminovich, and Christian P. Robert, 2020</p><p>A Conceptual Introduction to Hamiltonian Monte Carlo.
Michael Betancourt.
arXiv:1701.02434</p><h3 id=session-10-time-series-and-survival-analysis>Session 10: Time series and survival analysis</h3><p>LonGP: an additive Gaussian process regression model for longitudinal study designs.
Nature Communications, 10:1798.
Lu Cheng, Siddharth Ramchandran, Tommi Vatanen, Niina Lietzen, Riitta Lahesmaa, Aki Vehtari, and Harri Lähdesmäki (2019).</p><p>An interpretable probabilistic machine learning method for heterogeneous longitudinal studies.
Juho Timonen, Henrik Mannerström, Aki Vehtari, Harri Lähdesmäki (2019).
arXiv:1912.03549. .</p><p>Bayesian Survival Analysis Using the rstanarm R Package.
Samuel L. Brilleman, Eren M. Elci, Jacqueline Buros Novik, Rory Wolfe</p><p>Probabilistic Solutions To Ordinary Differential Equations As Non-Linear Bayesian Filtering: A New Perspective.
Filip Tronarp, Hans Kersting, Simo Särkkä, Philipp Hennig (2019).
Statistics and Computing.</p><p>A hierarchical Ornstein-Uhlenbeck model for stochastic time series analysis.
Ville Laitinen and Leo Lahti.
Advances in Intelligent Data Analysis XVII. Lecture Notes in Computer Science 11191., Springer, India, 2018. Conference proceedings. <a href=https://openresearchlabs.github.io/publications/papers/2018-Laitinen-IDA.pdf>https://openresearchlabs.github.io/publications/papers/2018-Laitinen-IDA.pdf</a></p><h3 id=session-11-gaussian-processes>Session 11: Gaussian processes</h3><p>Additive multivariate Gaussian processes for joint species distribution modeling with heterogeneous data.
J Vanhatalo, M Hartmann, L Veneranta.
Bayesian Analysis</p><p>Boettiger C, Mangel M, Munch S.
Avoiding tipping points in fisheries management through Gaussian process dynamic programming.
Proc Biol Sci. 2015;282(1801):20141631. doi:10.1098/rspb.2014.1631</p><p>Gaussian process modeling in approximate Bayesian computation to estimate horizontal gene transfer in bacteria.
Marko Järvenpää, Michael Gutmann, Aki Vehtari and Pekka Marttinen (2018).
The Annals of Applied Statistics, 12(4):2228-2251.
arXiv:1610.06462.</p><p>Deep Gaussian Processes.
Andreas C. Damianou, Neil D. Lawrence.
<a href=https://arxiv.org/abs/1211.0358>https://arxiv.org/abs/1211.0358</a></p><h3 id=session-12-bayesian-workflow>Session 12: Bayesian workflow</h3><p>Visualization in Bayesian workflow.
Journal of the Royal Statistical Society Series A, 182(2):389-402.
Jonah Gabry, Daniel Simpson, Aki Vehtari, Michael Betancourt, and Andrew Gelman (2019).
arXiv:1709.01449.</p><p>Toward a principled Bayesian workflow in cognitive science.
Daniel J. Schad, Michael Betancourt, Shravan Vasishth.
<a href=https://arxiv.org/abs/1904.12765>https://arxiv.org/abs/1904.12765</a></p><p>Validating Bayesian Inference Algorithms with Simulation-Based Calibration.
Sean Talts, Michael Betancourt, Daniel Simpson, Aki Vehtari, Andrew Gelman.
<a href=https://arxiv.org/pdf/1804.06788.pdf>https://arxiv.org/pdf/1804.06788.pdf</a></p><p>Jarno Lintusaari, Henri Vuollekoski, Antti Kangasrääsiö, Kusti Skytén, Marko Järvenpää, Michael Gutmann, Aki Vehtari, Jukka Corander, and Samuel Kaski (2018).
ELFI: Engine for Likelihood Free Inference.
In Journal of Machine Learning Research, 19(16):1-7, 2018.
arXiv:1708.00707</p><h3 id=session-13-deep-learning-neural-nets-and-autoencoders>Session 13: Deep learning, neural nets, and autoencoders</h3><p>Learning representations of microbe–metabolite interactions.
JT Morton, AA Aksenov, LF Nothias, JR Foulds, RA Quinn, MH Badri,
Nature methods 16 (12), 1306-1314</p><p>An Introduction to Variational Autoencoders.
Diederik P. Kingma, Max Welling.
<a href=https://arxiv.org/abs/1906.02691>https://arxiv.org/abs/1906.02691</a></p><p>Bayesian GAN.
Yunus Saatchi, Andrew Gordon Wilson.
<a href=https://arxiv.org/abs/1705.09558>https://arxiv.org/abs/1705.09558</a></p><p>Deep Learning: A Bayesian Perspective. Nicholas Polson, Vadim Sokolov.
<a href=https://arxiv.org/abs/1706.00473>https://arxiv.org/abs/1706.00473</a></p><p>Towards Bayesian Deep Learning: A Survey.
Hao Wang, Dit-Yan Yeung.
<a href=https://arxiv.org/abs/1604.01662>https://arxiv.org/abs/1604.01662</a></p><h3 id=session-14-applications>Session 14: Applications</h3><p>Multidomain analyses of a longitudinal human microbiome intestinal cleanout perturbation experiment.
Julia Fukuyama, Laurie Rumker, Kris Sankaran, Pratheepa Jeganathan, Les Dethlefsen, David A. Relman, Susan P. Holmes (2017)
PLOS Computational Biology, August 2017.</p><p>Fast hierarchical Bayesian analysis of population structure.
Tonkin-Hill, Gerry; Lees, John A; Bentley, Stephen D; Frost, Simon DW & Corander, Jukka (2019).
Nucleic Acids Research. 47(11),5539-5549.
doi: 10.1093/nar/gkz361</p><p>Additive multivariate Gaussian processes for joint species distribution modeling with heterogeneous data.
Jarno Vanhatalo†, Marcelo Hartmann‡and Lari Veneranta.
Bayesian analysis 2019.
<a href=https://arxiv.org/pdf/1809.02432.pdf>https://arxiv.org/pdf/1809.02432.pdf</a></p><h3 id=session-15-wrap-up>Session 15: Wrap-up</h3><h2 id=other-sources>Other sources</h2><p>You can also consider topics from the following books:</p><ul><li><a href=https://xcelab.net/rm/statistical-rethinking/>Statistical rethinking</a> (Richard McElreath; CRC Press)</li><li><a href=http://www.stat.columbia.edu/~gelman/book/>Bayesian Data Analysis</a> (Gelman et al. Chapman & Hall / CRC)</li><li><a href=https://www.springer.com/gp/book/9780387310732>Pattern recognition and Machine Learning</a> (Bishop; Springer)</li><li><a href=https://www.springer.com/gp/book/9780387848570>The Elements of Statistical Learning</a> (Hastie et al; Springer)</li><li><a href=https://www.amazon.com/Networks-Crowds-Markets-Reasoning-Connected/dp/0521195330>Networks, Crowds, and Markets</a> (Easley & Kleinberg; Cambridge University Press)</li></ul></div></div><div class=col-md-3></div></div></div></div><footer id=footer><div class=container><div class="col-md-4 col-sm-6"><h4>About us</h4><p>Turku Data Science Group.</p><hr class="hidden-md hidden-lg hidden-sm"></div><div class="col-md-4 col-sm-6"></div><div class="col-md-4 col-sm-6"><h4>Contact</h4><div class=social><script type=text/javascript>console.log("https://github.com/openresearchlabs/")</script><a href=https://github.com/openresearchlabs/ target=_blank style=opacity:1><i class='fab fa-2x fa-github'></i></a>
<script type=text/javascript>console.log("https://twitter.com/openreslabs")</script><a href=https://twitter.com/openreslabs target=_blank style=opacity:1><i class='fab fa-2x fa-twitter'></i></a>
<script type=text/javascript>console.log("mailto:leo.lahti@utu.fi")</script><a href=mailto:leo.lahti@utu.fi target=_blank style=opacity:1><i class='fas fa-2x fa-envelope'></i></a></div><p class=text-uppercase><strong>Turku Data Science Group</strong><br>University of Turku<br>Department of Computing<br>Turku<br>Finland<br><strong></strong></p><a href=/contact class="btn btn-small btn-template-main">Go to contact page</a><hr class="hidden-md hidden-lg hidden-sm"></div></div></footer><div id=copyright><div class=container><div class=col-md-12><p class=pull-left>Copyright (c) 2020-2021, Leo Lahti; some rights reserved.</p><p class=pull-right>Template by <a href=https://bootstrapious.com/p/universal-business-e-commerce-template>Bootstrapious</a>.
Ported to Hugo by <a href=https://github.com/devcows/hugo-universal-theme>DevCows</a>.</p></div></div></div></div><script async src="https://www.googletagmanager.com/gtag/js?id=G-3HREPRC83G"></script><script>var dnt,doNotTrack=!1;if(!1&&(dnt=navigator.doNotTrack||window.doNotTrack||navigator.msDoNotTrack,doNotTrack=dnt=="1"||dnt=="yes"),!doNotTrack){window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments)}gtag("js",new Date),gtag("config","G-3HREPRC83G")}</script><script src=//code.jquery.com/jquery-3.1.1.min.js integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin=anonymous></script><script src=//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js integrity=sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa crossorigin=anonymous></script><script src=//cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js></script><script src=//cdnjs.cloudflare.com/ajax/libs/waypoints/4.0.1/jquery.waypoints.min.js></script><script src=//cdnjs.cloudflare.com/ajax/libs/Counter-Up/1.0/jquery.counterup.min.js></script><script src=//cdnjs.cloudflare.com/ajax/libs/jquery-parallax/1.1.3/jquery-parallax.js></script><script src="//maps.googleapis.com/maps/api/js?v=3.exp"></script><script src=/js/hpneo.gmaps.js></script><script src=/js/gmaps.init.js></script><script src=/js/front.js></script><script src=/js/owl.carousel.min.js></script></body></html>