<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=robots content="all,follow"><meta name=googlebot content="index,follow,snippet,archive"><meta name=viewport content="width=device-width,initial-scale=1"><title>Research</title>
<meta name=author content="DevCows"><meta name=keywords content="devows,hugo,go,microbiome"><meta name=description content="Microbiome data science"><meta name=generator content="Hugo 0.140.1"><link href='//fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,500,700,800' rel=stylesheet type=text/css><link rel=stylesheet href=//use.fontawesome.com/releases/v5.11.2/css/all.css><link rel=stylesheet href=//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css integrity=sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u crossorigin=anonymous><link href=/css/animate.css rel=stylesheet><link href=/css/style.default.css rel=stylesheet id=theme-stylesheet><link href=/css/custom.css rel=stylesheet><!--[if lt IE 9]><script src=https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js></script><script src=https://oss.maxcdn.com/respond/1.4.2/respond.min.js></script><![endif]--><link rel="shortcut icon" href=/img/favicon.ico type=image/x-icon><link rel=apple-touch-icon href=/img/apple-touch-icon.png><link href=/css/owl.carousel.css rel=stylesheet><link href=/css/owl.theme.css rel=stylesheet><link rel=alternate href=http://datascience.utu.fi/index.xml type=application/rss+xml title="Turku Data Science Group"><meta name=twitter:card content="summary"><meta name=twitter:site content="@GoHugoIO"><meta name=twitter:title content="Research"><meta name=twitter:description content="Microbiome data science"><script async src="https://www.googletagmanager.com/gtag/js?id=G-3FJ60G61DS"></script><script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments)}gtag("js",new Date),gtag("config","G-3FJ60G61DS"),console.log("Test Google Analytics")</script><script type=text/javascript>_mtm=window._mtm=window._mtm||[],_mtm.push({"mtm.startTime":(new Date).getTime(),event:"mtm.Start"});var _mtm,d=document,g=d.createElement("script"),s=d.getElementsByTagName("script")[0];g.type="text/javascript",g.async=!0,g.src="https://analytics.utu.fi/js/container_5AYuB7oa.js",s.parentNode.insertBefore(g,s)</script></head><body><div id=all><header class=navbar-affixed-top data-spy=affix data-offset-top=62><div class="navbar navbar-default yamm" role=navigation style=padding-left:0 id=navbar><div class=container><div class=navbar-header><a class="navbar-brand home" href=/><img src=/img/logopic/university_turku.png alt="Research logo" class="hidden-xs hidden-sm university-logo">
<span class=sr-only>Research - go to homepage</span></a><div class=navbar-buttons><button type=button class="navbar-toggle btn-template-main" data-toggle=collapse data-target=#navigation>
<span class=sr-only>Toggle Navigation</span>
<i class="fas fa-align-justify"></i></button></div><a href=/><div class=title>Turku Data Science Group</div></a></div><div class="navbar-collapse collapse" id=navigation><ul class="nav navbar-nav navbar-right"><li class=dropdown><li><a href=/>Home</a></li></li><li class="dropdown active"><li class=dropdown-list><a class=dropbtn href=/research-topics>Research</a></li></li><li class=dropdown><li class=dropdown-list><a class=dropbtn href=/edu-topics>Edu</a></li><li class=dropdown><li><a href=/code/>Code</a></li></li><li class=dropdown><li><a href=/team/>Team</a></li></li><li class=dropdown><li><a href=/contact/>Contact</a></li></li></ul></div><div class="collapse clearfix" id=search><form class=navbar-form role=search><div class=input-group><input type=text class=form-control placeholder=Search>
<span class=input-group-btn><button type=submit class="btn btn-template-main"><i class="fas fa-search"></i></button></span></div></form></div></div></div></header><div id=heading-breadcrumbs><div class=container><div class=row><div class=col-md-12><h1>Research</h1></div></div></div></div><div id=content><div class=container><div class=row><div class=col-md-9 id=blog-post><div class=research-intro><p><br><b>Our research focuses on computational analysis of
complex natural and social systems.</b> There is a great demand for
targeted computational techniques to extract information and insights
from rich data collections based on clever combinations of human and
machine intelligence. We blend elements from fields such as machine
learning/AI, probabilistic programming, statistical ecology, and data
science, and drive open developer communities that help to translate
latest theoretical advances into accessible methods to inform
modeling, experimentation, and decision-making. For a full list of publications check <a href=http://datascience.utu.fi/research-topics/>this page</a>.</div><div id=post-content><h3 id=microbiome-data-science-selected-examples>Microbiome data science: selected examples</h3><p><b>Taxonomic signatures of cause-specific mortality risk in human gut microbiome</b><br>Salosensaari*
A,
Laitinen*
V,
Havulinna
A,
Meric
G,
Cheng
S,
Perola
M,
Valsta
L,
Alfthan
G,
Inouye
M,
Watrous
J,
Long
T,
Salido
R,
Sanders
K,
Brennan
C,
Humphrey
G,
Sanders
J,
Jain
M,
Jousilahti
P,
Salomaa
V,
Knight
R,
Lahti*
L & 
Niiranen*
T.<br>Nature Communications 12,
2021<br><a href=https://doi.org/10.1038/s41467-021-22962-y>10.1038/s41467-021-22962-y</a></p><p><b>Statistical and machine learning techniques in human microbiome studies: Contemporary challenges and solutions</b><br>Moreno-Indias
I,
Lahti
L,
Nedyalkova
M,
Elbere
I,
Roshchupkin
G,
Adilovic
M,
Aydemir
O,
Bakir-Gungor
B,
Pau
E,
D’Elia
D,
Desai
M,
Falquet
L,
Gundogdu
A,
Hron
K,
Klammsteiner
T,
Lopes
M,
Zambrano
L,
Marques
C,
Mason
M,
May
P,
Pašić
L,
Pio
G,
Pongor
S,
Promponas
V,
Przymus
P,
Sáez-Rodríguez
J,
Sampri
A,
Shigdel
R,
Stres
B,
Suharoschi
R,
Truu
J,
Truică
C,
Vilne
B,
Vlachakis
D,
Yılmaz
E,
Zeller
G,
Zomer
A,
Gómez-Cabrero
D & 
Claesson
M.<br>Frontiers in Microbiology 12,
2021<br><a href=https://doi.org/10.3389/fmicb.2021.635781>10.3389/fmicb.2021.635781</a></p><p><b>Microbiome data science in the <i>SummarizedExperiment</i> universe</b><br>Lahti
L,
Ernst
F,
Shetty
S & 
others.<br>F1000Research 10(748),
2021<br>(slides); version 1; not peer reviewed<br><a href=https://doi.org/10.7490/f1000research.1118712.1>10.7490/f1000research.1118712.1</a></p><p><b>Modeling spatial patterns in host-associated microbial communities</b><br>Ruuskanen
M,
Sommeria-Klein
G,
Havulinna
A,
Niiranen
T & 
Lahti
L.<br>Environmental Microbiology 23(5),
2021<br><a href=https://doi.org/10.1111/1462-2920.15462>10.1111/1462-2920.15462</a></p><p><b>Microbial communities as dynamical systems</b><br>Gonze
D,
Coyte
K,
Lahti
L & 
Faust
K.<br>Current Opinion in Microbiology 44,
2018<br><a href=https://doi.org/10.1016/j.mib.2018.07.004>10.1016/j.mib.2018.07.004</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2018-Gonze.pdf>PDF</a></p><p><b>Multi-stability and the origin of microbial community types</b><br>Gonze
D,
Lahti
L,
Raes
J & 
Faust
K.<br>ISME Journal 11,
2017<br><a href=https://doi.org/10.1038/ismej.2017.60>10.1038/ismej.2017.60</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2017-Faust-ISME.pdf>PDF</a></p><p><b>Signatures of ecological processes in microbial community time series</b><br>Faust
K,
Bauchinger
F,
Laroche
B,
Buyl
S,
Lahti
L,
Washburne
A,
Gonze
D & 
Widder
S.<br>Microbiome 6(120),
2018<br><a href=https://doi.org/10.1186/s40168-018-0496-2>10.1186/s40168-018-0496-2</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2018-Faust-Microbiome.pdf>PDF</a></p><p><b>Linking statistical and ecological theory: Hubbell’s unified neutral theory of biodiversity as a hierarchical dirichlet process</b><br>Harris
K,
Parsons
T,
Ijaz
U,
Lahti
L,
Holmes
I & 
Quince
C.<br>Proceedings of the IEEE 105(3),
2017<br><a href=https://doi.org/10.1109/JPROC.2015.2428213>10.1109/JPROC.2015.2428213</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2017-Harris-IEEE.pdf>PDF</a></p><h3 id=more-microbiome-data-science-and-life-science-applications>More microbiome data science and life science applications</h3><p><b>LimROTS: A hybrid method integrating empirical bayes and reproducibility-optimized statistics for robust analysis of proteomics and metabolomics data.</b><br>Mostafa
A,
Lahti
L,
Jeba
A & 
Coffey
E.<br>Bioconductor open source software for Bioinformatics
2024<br><a href=https://doi.org/10.18129/B9.bioc.LimROTS>10.18129/B9.bioc.LimROTS</a>
|
<a href=https://bioconductor.org/packages/LimROTS>URL</a></p><p><b>Examining the healthy human microbiome concept</b><br>Joos
R,
Boucher
K,
Lavelle
A,
Arumugam
M,
Blaser
M,
Claesson
M,
Clarke
G,
Cotter
P,
Sordi
L,
Dominguez-Bello
M,
Dutilh
B,
Ehrlich
S,
Ghosh
T,
Hill
C,
Junot
C,
Lahti
L,
Lawley
T,
Licht
T,
Maguin
E,
Makhalanyane
T,
Marchesi
J,
Matthijnssens
J,
Raes
J,
Ravel
J,
Salonen
A,
Scanlan
P,
Shkoporov
A,
Stanton
C,
Thiele
I,
Tolstoy
I,
Walter
J,
Yang
B,
Yutin
N,
Zhernakova
A,
Zwart
H,
,
Doré
J & 
Ross
R.<br>Nature Reviews Microbiology
2024<br><a href=https://doi.org/10.1038/s41579-024-01107-0>10.1038/s41579-024-01107-0</a></p><p><b>Regular exercise training induces more changes on intestinal glucose uptake from blood and microbiota composition in leaner compared to heavier individuals in monozygotic twins discordant for BMI</b><br>Lietzén
M,
Guzzardi
M,
Ojala
R,
Hentilä
J,
Heiskanen
M,
Honkala
S,
Lautamäki
R,
Löyttyniemi
E,
Kirjavainen
A,
Rajander
J,
Malm
T,
Lahti
L,
Rinne
J,
Pietiläinen
K,
Iozzo
P & 
Hannukainen
J.<br>Nutrients 16,
2024<br><a href=https://doi.org/10.3390/nu16203554>10.3390/nu16203554</a></p><p><b>Learning and teaching biological data science in the bioconductor community</b><br>Drnevich
J,
Tan
F,
Almeida-Silva
F,
Castelo
R,
Culhane
A,
Davis
S,
Doyle
M,
Holmes
S,
Lahti
L,
Mahmoud
A,
Nishida
K,
Ramos
M,
Rue-Albrecht
K,
Shih
D,
Gatto
L & 
Soneson
C.<br>arXiv
2024<br><a href=https://doi.org/10.48550/arXiv.2410.01351>10.48550/arXiv.2410.01351</a></p><p><b>Effects of obesity and exercise on hepatic and pancreatic lipid content and glucose metabolism: PET-studies in twins discordant for BMI</b><br>Lietzén
M,
Mari
A,
Ojala
R,
Hentilä
J,
Koskensalo
K,
Lautamäki
R,
Löyttyniemi
E,
Parkkola
R,
Saunavaara
V,
Kirjavainen
A,
Rajander
J,
Malm
T,
Lahti
L,
Rinne
J,
Pietiläinen
K,
Iozzo
P & 
Hannukainen
J.<br>Biomolecules 14(9),
2024<br><a href=https://doi.org/10.3390/biom14091070>10.3390/biom14091070</a></p><p><b>A cohort study in family triads: Impact of gut microbiota composition and early life exposures on intestinal resistome during the first two years of life</b><br>Jokela
R,
Pärnänen
K,
Ponsero
A,
Lahti
L,
Kolho
K,
Vos
W & 
Salonen
A.<br>Gut Microbes 16(1),
2024<br><a href=https://doi.org/10.1080/19490976.2024.2383746>10.1080/19490976.2024.2383746</a></p><p><b>Associations between gut microbiota and incident fractures in the FINRISK cohort</b><br>Grahnemo
L,
Kambur
O,
Lahti
L,
Jousilahti
P,
Niiranen
T,
Knight
R,
Salomaa
V,
Havulinna
A & 
Ohlsson
C.<br>npj Biofilms and Microbiomes 10(69),
2024<br><a href=https://doi.org/10.1038/s41522-024-00530-8>10.1038/s41522-024-00530-8</a></p><p><b>Multikingdom oral microbiome interactions in early-onset cryptogenic ischemic stroke</b><br>Manzoor
M,
Leskelä
J,
Pietiäinen
M,
Martinez-Majander
N,
Ylikotila
P,
Könönen
E,
Niiranen
T,
Lahti
L,
Sinisalo
J,
Putaala
J,
Pussinen
P & 
Paju
S.<br>ISME Communications 4(1),
2024<br><a href=https://doi.org/10.1093/ismeco/ycae088>10.1093/ismeco/ycae088</a></p><p><b>Association between butyrate-producing gut bacteria and the risk of infectious disease hospitalisation: Results from two observational, population-based microbiome studies</b><br>Kullberg
R,
Wikki
I,
Haak
B,
Kauko
A,
Galenkamp
H,
Peters-Sengers
H,
Butler
J,
Havulinna
A,
Palmu
J,
McDonald
D,
Benchraka
C,
Abdel-Aziz
M,
Prins
M,
Maitland van der Zee
A,
Born
B,
Jousilahti
P,
Vos
W,
Salomaa
V,
Knight
R,
Lahti
L,
Nieuwdorp
M,
Niiranen
T & 
Wiersinga
W.<br>The Lancet Microbe
2024<br><a href=https://doi.org/https://doi.org/10.1016/S2666-5247%2824%2900079-X>https://doi.org/10.1016/S2666-5247(24)00079-X</a></p><p><b>Biochemical analyses can complement sequencing-based ARG load monitoring: A case study in indian hospital sewage networks</b><br>Bhanushali
S,
Parnanen
K,
Mongad
D,
Dhotre
D & 
Lahti
L.<br>medRxiv
2024<br><a href=https://doi.org/10.1101/2024.05.31.24308262>10.1101/2024.05.31.24308262</a></p><p><b>A concept for international societally relevant microbiology education and microbiology knowledge promulgation in society</b><br>Timmis
K,
Hallsworth
J,
McGenity
T,
Armstrong
R,
Colom
M,
Karahan
Z,
Chavarría
M,
Bernal
P,
Boyd
E,
Ramos
J,
Kaltenpoth
M,
Pruzzo
C,
Clarke
G,
López-Garcia
P,
Yakimov
M,
Perlmutter
J,
Greening
C,
Eloe-Fadrosh
E,
Verstraete
W,
Nunes
O,
Kotsyurbenko
O,
Nikel
P,
Scavone
P,
Lavigne
M,
Roux
F,
Timmis
J,
Parro
V,
Michán
C,
García
J,
Casadevall
A,
Payne
S,
Frey
J,
Koren
O,
Prosser
J,
Lahti
L,
Lal
R,
Anand
S,
Sood
U,
Offre
P,
Bryce
C,
Mswaka
A,
Jores
J,
Kaçar
B,
Blank
L,
Maaßen
N,
Pope
P,
Banciu
H,
Armitage
J,
Lee
S,
Wang
F,
Makhalanyane
T,
Gilbert
J,
Wood
T,
Vasiljevic
B,
Soberón
M,
Udaondo
Z,
Rojo
F,
Tamang
J,
Giraud
T,
Ropars
J,
Ezeji
T,
Müller
V,
Danbara
H,
Averhoff
B,
Sessitsch
A,
Partida-Martínez
L,
Huang
W,
Molin
S,
Junier
P,
Amils
R,
Wu
X,
Ron
E,
Erten
H,
Martinis
E,
Rapoport
A,
Öpik
M,
Pokatong
W,
Stairs
C,
Amoozegar
M & 
Serna
J.<br>Microbial Biotechnology 17(5),
2024<br><a href=https://doi.org/10.1111/1751-7915.14456>10.1111/1751-7915.14456</a></p><p><b>Microbiome confounders and quantitative profiling challenge predicted microbial targets in colorectal cancer development</b><br>Tito
R,
Verbandt
S,
Vazquez
M,
Lahti
L,
Verspecht
C,
Lloréns-Rico
V,
Vieira-Silva
S,
Arts
J,
Falony
G,
Dekker
E,
Reumers
J,
Tejpar
S & 
Raes
J.<br>Nature Medicine
2024<br><a href=https://doi.org/10.1038/s41591-024-02963-2>10.1038/s41591-024-02963-2</a></p><p><b>Fecal microbiota profiles of growing pigs and their relation to growth performance</b><br>König
E,
Beasley
S,
Heponiemi
P,
Kivinen
S,
Räkköläinen
J,
Salminen
S,
Collado
M,
Borman
T,
Lahti
L,
Piirainen
V,
Valros
A & 
Heinonen
M.<br>PLoS One 19,
2024<br><a href=https://doi.org/10.1371/journal.pone.0302724>10.1371/journal.pone.0302724</a></p><p><b>Elementary methods provide more replicable results in microbial differential abundance analysis</b><br>Pelto
J,
Auranen
K,
Kujala
J & 
Lahti
L.<br>arXiv
2024<br><a href=https://doi.org/10.48550/arXiv.2404.02691>10.48550/arXiv.2404.02691</a>
|
<a href=https://arxiv.org/abs/2404.02691>URL</a></p><p><b>Shotgun metagenomic analysis of the oral microbiome in gingivitis: A nested case-control study</b><br>Manzoora
M,
Leskelä
J,
Pietiäinen
M,
Martinez-Majander
N,
Könönen
E,
Niiranen
T,
Lahti
L,
Sinisalo
J,
Putaala
J,
Pussinen
P & 
Paju
S.<br>Journal of Oral Microbiology (ZJOM) 16,
2024<br><a href=https://doi.org/10.1080/20002297.2024.2330867>10.1080/20002297.2024.2330867</a></p><p><b>Integration of polygenic and gut metagenomic risk prediction for common diseases</b><br>Liu
Y,
Ritchie
S,
Teo
S,
Ruuskanen
M,
Kambur
O,
Zhu
Q,
Sanders
J,
Vázquez-Baeza
Y,
Verspoor
K,
Jousilahti
P,
Lahti
L,
Niiranen
T,
Salomaa
V,
Havulinna
A,
Knight
R,
Méric
G & 
Inouye
M.<br>Nature Aging
2024<br><a href=https://doi.org/10.1038/s43587-024-00590-7>10.1038/s43587-024-00590-7</a></p><p><b>Gut microbiome-derived bacterial extracellular vesicles in patients with solid tumours</b><br>Mishra
S,
Tejesvi
M,
Hekkala
J,
Turunen
J,
Kandikanti
N,
Kaisanlahti
A,
Suokas
M,
Leppä
S,
Vihinen
P,
Kuitunen
H,
Sunela
K,
Koivunen
J,
Jukkola
A,
Kalashnikov
I,
Auvinen
P,
Kääriäinen
O,
Medina
T,
Medina
O,
Saarnio
J,
Meriläinen
S,
Rautio
T,
Aro
R,
Häivälä
R,
Suojanen
J,
Laine
M,
Erawijattari
P,
Lahti
L,
Karihtala
P,
Ruuska
T & 
Reunanen
J.<br>Journal of Advanced Research
2024<br>Available online 7 March 2024<br><a href=https://doi.org/10.1016/j.jare.2024.03.003>10.1016/j.jare.2024.03.003</a></p><p><b>Association of long-term habitual dietary fiber intake since infancy with gut microbiota composition in young adulthood</b><br>Heiskanen
M,
Aatsinki
A,
Hakonen
P,
Kartiosuo
N,
Munukka
E,
Lahti
L,
Keskitalo
A,
Huovinen
P,
Niinikoski
H,
Viikari
J,
Rönnemaa
T,
Lagström
H,
Jula
A,
Raitakari
O,
Rovio
S & 
Pahkala
K.<br>The Journal of Nutrition 154,
2024<br>Available online 12 January 2024<br><a href=https://doi.org/10.1016/j.tjnut.2024.01.008>10.1016/j.tjnut.2024.01.008</a></p><p><b>Bone marrow metabolism is affected by body weight and response to exercise training varies according to anatomical location</b><br>Ojala
R,
Hentilä
J,
Lietzén
M,
Arponen
M,
Heiskanen
M,
Honkala
S,
Virtanen
H,
Koskensalo
K,
Lautamäki
R,
Löyttyniemi
E,
Parkkola
R,
Heinonen
O,
Malm
T,
Lahti
L,
Rinne
J,
Eskola
O,
Rajander
J,
Pietiläinen
K,
Kaprio
J,
Ivaska
K & 
Hannukainen
J.<br>Diabetes, Obesity and Metabolism 16,
2024<br>First published 11 October 2023<br><a href=https://doi.org/10.1111/dom.15311>10.1111/dom.15311</a></p><p><b>Daily skin-to-skin contact alters microbiota development in healthy full-term infants</b><br>Eckermann
H,
Meijer
J,
Cooijmans
K,
Lahti
L & 
Weerth
C.<br>Gut Microbes 16(2295403, 1),
2024<br><a href=https://doi.org/10.1080/19490976.2023.2295403>10.1080/19490976.2023.2295403</a></p><p><b>Fewer culturable Lactobacillaceae species identified in faecal samples of pigs performing manipulative behaviour</b><br>König
E,
Heponiemi
P,
Kivinen
S,
Räkköläinen
J,
Beasley
S,
Borman
T,
Collado
M,
Hukkinen
V,
Junnila
J,
Lahti
L,
Norring
M,
Piirainen
V,
Salminen
S,
Heinonen
M & 
Valros
A.<br>Scientific Reports 14(132),
2024<br><a href=https://doi.org/10.1038/s41598-023-50791-0>10.1038/s41598-023-50791-0</a></p><p><b>Infant gut microbiota and negative and fear reactivity</b><br>Huovinen
V,
Aatsinki
A,
Kataja
E,
Munukka
E,
Keskitalo
A,
Lamichhane
S,
Raunioniemi
P,
Bridgett
D,
Lahti
L,
O’Mahony
S,
Dickens
A,
Korja
R,
Karlsson
H,
Nolvi
S & 
Karlsson
L.<br>Development and Psychopathology
2023<br><a href=https://doi.org/10.1017/S0954579423001396>10.1017/S0954579423001396</a></p><p><b>A toolbox of machine learning software to support microbiome analysis</b><br>Marcos-Zambrano
L,
López-Molina
V,
Bakir-Gungor
B,
Frohme
M,
Karaduzovic-Hadziabdic
K,
Klammsteiner
T,
Ibrahimi
E,
Lahti
L,
Loncar-Turukalo
T,
Dhamo
X,
Simeon
A,
Nechyporenko
A,
Pio
G,
Przymus
P,
Sampri
A,
Trajkovik
V,
Lacruz-Pleguezuelos
B,
Aasmets
O,
Araujo
R,
Anagnostopoulos
I,
Aydemir
Ö,
Berland
M,
Calle
M,
Ceci
M,
Duman
H,
Gündoğdu
A,
Havulinna
A,
Bra
K,
Kalluci
E,
Karav
S,
Lode
D,
Lopes
M,
May
P,
Nap
B,
Nedyalkova
M,
Paciência
I,
Pasic
L,
Pujolassos
M,
Shigdel
R,
Susín
A,
Thiele
I,
Truică
C,
Wilmes
P,
Yilmaz
E,
Yousef
M,
Claesson
M,
Truu
J & 
Santa Pau
E.<br>Frontiers in Microbiology 14,
2023<br><a href=https://doi.org/10.3389/fmicb.2023.1250806>10.3389/fmicb.2023.1250806</a>
|
<a href=https://www.frontiersin.org/articles/10.3389/fmicb.2023.1250806>URL</a></p><p><b>Role of Gut Microbiota in Statin-Associated New-Onset Diabetes—a Cross-Sectional and Prospective Analysis of the FINRISK 2002 Cohort</b><br>Koponen
K,
Kambur
O,
Joseph
B,
Ruuskanen
M,
Jousilahti
P,
Salido
R,
Brennan
C,
Jain
M,
Meric
G,
Inouye
M,
Lahti
L,
Niiranen
T,
Havulinna
A,
Knight
R & 
Salomaa
V.<br>Arteriosclerosis, Thrombosis, and Vascular Biology
2023<br><a href=https://doi.org/10.1161/ATVBAHA.123.319458>10.1161/ATVBAHA.123.319458</a></p><p><b>What are patterns of rise and decline?</b><br>Raulo
A,
Rojas
A,
Kröger
B,
Laaksonen
A,
Orta
C,
Nurmio
S,
Peltoniemi
M,
Lahti
L & 
Žliobaitė
I.<br>The Royal Society of Open Science 10,
2023<br><a href=https://doi.org/10.1098/rsos.230052>10.1098/rsos.230052</a></p><p><b>Maternal microbiota communicates with the fetus through microbiota-derived extracellular vesicles</b><br>Kaisanlahti
A,
Turunen
J,
Byts
N,
Samoylenko
A,
Bart
G,
Virtanen
N,
Tejesvi
M,
Zhyvolozhnyi
A,
Sarfraz
S,
Kumpula
S,
Hekkala
J,
Salmi
S,
Will
O,
Korvala
J,
Paalanne
N,
Erawijantari
P,
Suokas
M,
Medina
T,
Vainio
S,
Medina
O,
Lahti
L,
Tapiainen
T & 
Reunanen
J.<br>BMC Microbiome 249,
2023<br><a href=https://doi.org/10.1186/s40168-023-01694-9>10.1186/s40168-023-01694-9</a>
|
<a href=https://rdcu.be/dq2aL>URL</a></p><p><b>Microbiome-based risk prediction in incident heart failure: A community challenge</b><br>Erawijantari
P,
Kartal
E,
Liñares-Blanco
J,
Laajala
T,
Feldman
L,
Challenge
T,
Communities
M,
Carmona-Saez
P,
Shigdel
R,
Marcus
,
Claesson
J,
Bertelsen
R,
Gomez-Cabrero
D,
Minot
S,
Albrecht
J,
Chung
V,
Inouye
M,
Jousilahti
P,
Schultz
J,
Friederich
H,
Knight
R,
Salomaa
V,
Niiranen
T,
Havulinna
A,
Saez-Rodriguez
J,
Levinson
R & 
Lahti
L.<br>medRxiv preprint
2023<br><a href=https://doi.org/10.1101/2023.10.12.23296829>10.1101/2023.10.12.23296829</a>
|
<a href=https://www.medrxiv.org/content/10.1101/2023.10.12.23296829v1>URL</a></p><p><b>Impacts of maternal microbiota and microbial metabolites on fetal intestine, brain, and placenta</b><br>Husso
A,
Pessa-Morikawa
T,
Mikael
V,
Kärkkäinen
O,
Kwon
H,
Lahti
L,
Iivanainen
A,
Hanhineva
K & 
Niku
M.<br>BMC Biology(207),
2023<br><a href=https://doi.org/10.1186/s12915-023-01709-9>10.1186/s12915-023-01709-9</a></p><p><b>Advancing microbiome research with machine learning: key findings from the ML4Microbiome COST action</b><br>D’Elia
D,
Truu
J,
Lahti
L,
Berland
M,
Papoutsoglou
G,
Ceci
M,
Zomer
A,
Lopes
M,
Ibrahimi
E,
Gruca
A,
Nechyporenko
A,
Frohme
M,
Carrillo-de Santa Pau
T,
Marcos-Zambrano
L,
Hron
K,
Pio
G,
Simeon
A,
Suharoschi
R,
Moreno-Indias
I,
Temko
A,
Nedyalkova
M,
Apostol
E,
Truică
C,
Shigdel
R,
Hasić-Telalović
J,
Bongcam-Rudloff
E,
Przymus
P,
Jordamović
N,
Falquet
L,
Tarazona
S,
Sampri
A,
Isola
G,
Pérez-Serrano
D,
Trajkovik
V,
Klucar
L,
Loncar-Turukalo
T,
Havulinna
A,
Jansen
C & 
Claesson
R.<br>Frontiers in Microbiology 14,
2023<br><a href=https://doi.org/10.3389/fmicb.2023.1257002>10.3389/fmicb.2023.1257002</a></p><p><b>Greengenes2 unifies microbial data in a single reference tree</b><br>McDonald
D,
Jiang
Y,
Balaban
M,
Cantrell
K,
Zhu
Q,
Gonzalez
A,
Morton
J,
Nicolaou
G,
Parks
D,
Karst
S,
Albertsen
M,
Hugenholtz
P,
DeSantis
T,
Song
S,
Bartko
A,
Havulinna
A,
Jousilahti
P,
Cheng
S,
Inouye
M,
Niiranen
T,
Jain
M,
Salomaa
V,
Lahti
L,
Mirarab
S & 
Knight
R.<br>Nature Biotechnology
2023<br><a href=https://doi.org/10.1038/s41587-023-01845-1>10.1038/s41587-023-01845-1</a></p><p><b>miaSim: an R/Bioconductor package to easily simulate microbial community dynamics</b><br>Gao
Y,
Şimşek
Y,
Gheysen
E,
Borman
T,
Li
Y,
Lahti
L,
Faust
K & 
Garza
D.<br>Methods in Ecology and Evolution 14,
2023<br><a href=https://doi.org/10.1111/2041-210X.14129>10.1111/2041-210X.14129</a></p><p><b>Infant left amygdala volume is negatively associated with fecal microbiota diversity</b><br>Aatsinki
A,
Tuulari
J,
Munukka
E,
Lahti
L,
Keskitalo
A,
Kailanto
H,
Nolvi
S,
Scheinin
N,
Saunavaara
J,
Parkkola
R,
Lewis
J,
Hashempour
N,
Shulist
S,
Karlsson
L & 
Karlsson
H.<br>bioRxiv preprint
2023<br><a href=https://doi.org/10.1101/2023.04.23.537273>10.1101/2023.04.23.537273</a></p><p><b>Gut microbiome and atrial fibrillation-results from a large population-based study</b><br>Palmu
J,
Börschel
C,
Ortega-Alonso
A,
Lajos
M,
Inouye
M,
Jousilahti
P,
Salido
R,
Sanders
K,
Brennan
C,
Humphrey
G,
Sanders
J,
Gutmann
F,
Linz
D,
Salomaa
V,
Havulinna
A,
Forslund
S,
Knight
R,
Lahti
L,
Niiranen
T & 
Schnabel
R.<br>eBioMedicine
2023<br><a href=https://doi.org/10.1016/j.ebiom.2023.104583>10.1016/j.ebiom.2023.104583</a></p><p><b>Gut microbiota composition and function in pregnancy as determinants of prediabetes at two years’ postpartum</b><br>Houttu
N,
Benchraka
C,
Lotankar
M,
Muhli
E,
Niinikoski
H,
Lahti
L & 
Laitinen
K.<br>Acta Diabetologica
2023<br><a href=https://doi.org/10.1007/s00592-023-02064-5>10.1007/s00592-023-02064-5</a></p><p><b>The preterm gut microbiota and administration routes of different probiotics: A randomized controlled trial</b><br>Rahkola
E,
Rautava
S,
Hiltunen
H,
Ross
C,
Lahti
L & 
Isolauri
E.<br>Pediatric research
2023<br><a href=https://doi.org/10.1038/s41390-023-02560-y>10.1038/s41390-023-02560-y</a></p><p><b>The gut microbiome is a significant risk factor for future chronic lung disease</b><br>Liu
Y,
Teo
S,
Meric
G,
Tang
H,
Zhu
Q,
Sanders
J,
Vazquez-Baeza
Y,
Verspoor
K,
Vartiainen
V,
Jousilahti
P,
Lahti
L,
Niiranen
T,
Havulinna
A,
Knight
R,
Salomaa
V & 
Inouye
M.<br>Journal of Allergy and Clinical Immunology 151,
2023<br><a href=https://doi.org/10.1016/j.jaci.2022.12.810>10.1016/j.jaci.2022.12.810</a></p><p><b>Aberrations in the early pregnancy serum metabolic profile in women with prediabetes at two years postpartum</b><br>Muhli
E,
Benchraka
C,
Lotankar
M,
Houttu
N,
Niinikoski
H,
Lahti
L & 
Laitinen
K.<br>Metabolomics 19(29),
2023<br><a href=https://doi.org/10.1007/s11306-023-01994-z>10.1007/s11306-023-01994-z</a></p><p><b>Dealing with dimensionality: the application of machine learning to multi-omics data</b><br>Feldner-Busztin
D,
Firbas Nisantzis
P,
Edmunds
S,
Boza
G,
Racimo
F,
Gopalakrishnan
S,
Limborg
M,
Lahti
L & 
Polavieja
G.<br>Bioinformatics 39(2),
2023<br>btad021<br><a href=https://doi.org/10.1093/bioinformatics/btad021>10.1093/bioinformatics/btad021</a>
|
<a href=https://doi.org/10.1093/bioinformatics/btad021>URL</a></p><p><b>Multi-trait genetic analysis reveals clinically interpretable hypertension subtypes</b><br>Vaura
F,
Kim
H,
Udler
M,
,
Salomaa
V,
Lahti
L & 
Niiranen
T.<br>Circulation: Genomic and Precision Medicine 15(4),
2022<br><a href=https://doi.org/10.1161/CIRCGEN.121.003583>10.1161/CIRCGEN.121.003583</a></p><p><b>Comprehensive biomarker profiling of hypertension in 36 985 finnish individuals</b><br>Palmu
J,
Tikkanen
E,
Havulinna
A,
Vartiainen
E,
Lundqvist
A,
Ruuskanen
M,
Perola
M,
Ala-Korpela
M,
Jousilahti
P,
Würtz
P,
Salomaa
V,
Lahti
L & 
Niiranen
T.<br>Journal of Hypertension 40(3),
2022<br><a href=https://doi.org/10.1097/HJH.0000000000003051>10.1097/HJH.0000000000003051</a></p><p><b>Impacts of maternal microbiota and microbial metabolites on fetal intestine, brain and placenta</b><br>Husso
A,
Pessa-Morikawa
T,
Koistinen
V,
Karkkainen
O,
Lahti
L,
Iivanainen
A,
Hanhineva
K & 
Niku
M.<br>bioRxiv Cold Spring Harbor Laboratory,
2022<br><a href=https://doi.org/10.1101/2022.07.01.498433>10.1101/2022.07.01.498433</a>
|
<a href=https://www.biorxiv.org/content/early/2022/07/03/2022.07.01.498433>URL</a></p><p><b>The gut microbiome is a significant risk factor for future chronic lung disease</b><br>Liu
Y,
Teo
S,
Meric
G,
Tang
H,
Zhu
Q,
Sanders
J,
Vazquez-Baeza
Y,
Verspoor
K,
Vartiainen
V,
Jousilahti
P,
Lahti
L,
Niiranen
T,
Havulinna
A,
Knight
R,
Salomaa
V & 
Inouye
M.<br>medRxiv
2022<br><a href=https://doi.org/10.1101/2022.03.22.22272736>10.1101/2022.03.22.22272736</a></p><p><b>An infancy-onset 20-year dietary counselling intervention and gut microbiota composition in adulthood</b><br>Keskitalo
A,
Munukka
E,
Aatsinki
A,
Saleem
W,
Kartiosuo
N,
Lahti
L,
Huovinen
P,
Elo
L,
Pietilä
S,
Rovio
S,
Niinikoski
H,
Viikari
J,
Rönnemaa
T,
Lagström
H,
Jula
A,
Raitakari
O & 
Pahkala
K.<br>Nutrients 14(13),
2022<br><a href=https://doi.org/10.3390/nu14132667>10.3390/nu14132667</a></p><p><b>Towards standardized and reproducible research in skin microbiomes</b><br>Ruuskanen
M,
Vats
D,
Potbhare
R,
RaviKumar
A,
Munukka
E,
Ashma
R & 
Lahti
L.<br>Environmental Microbiology 24(9),
2022<br><a href=https://doi.org/10.1111/1462-2920.15945>10.1111/1462-2920.15945</a></p><p><b>Potential pathobionts in vaginal microbiota are affected by fish oil and/or probiotics intervention in overweight and obese pregnant women</b><br>Houttu
N,
Mokkala
K,
Saleem
W,
Virtanen
S,
Juhila
J,
Koivuniemi
E,
Pellonperä
O,
Tertti
K,
Luokola
P,
Sorsa
T,
Salonen
A,
Lahti
L & 
Laitinen
K.<br>Biomedicine & Pharmacotherapy 149,
2022<br><a href=https://doi.org/10.1016/j.biopha.2022.112841>10.1016/j.biopha.2022.112841</a>
|
<a href=https://www.sciencedirect.com/science/article/pii/S0753332222002293>URL</a></p><p><b>Skin microbiota diversity among genetically unrelated individuals of indian origin</b><br>Potbhare
R,
RaviKumar
A,
Munukka
E,
Lahti
L & 
Ashma
R.<br>PeerJ 10,
2022<br><a href=https://doi.org/10.7717/peerj.13075>10.7717/peerj.13075</a></p><p><b>Can gut microbiota throughout the first 10 years of life predict executive functioning in childhood?</b><br>Eckermann
H,
Ou
Y,
Lahti
L & 
Weerth
C.<br>Developmental Psychobiology 64(3),
2022<br><a href=https://doi.org/10.1002/dev.22226>10.1002/dev.22226</a></p><p><b>Phylogeny-aware analysis of metagenome community ecology based on matched reference genomes while bypassing taxonomy</b><br>Zhu
Q,
Huang
S,
Gonzalez
A,
McGrath
I,
McDonald
D,
Haiminen
N,
Armstrong
G,
Vázquez-Baeza
Y,
Yu
J,
Kuczynski
J,
Sepich-Poore
G,
Swafford
A,
Das
P,
Shaffer
J,
Lejzerowicz
F,
Belda-Ferre
P,
Havulinna
A,
Méric
G,
Niiranen
T,
Lahti
L,
Salomaa
V,
Kim
H,
Jain
M,
Inouye
M,
Gilbert
J & 
Knight
R.<br>mSystems 7,
2022<br><a href=https://doi.org/10.1128/msystems.00167-22>10.1128/msystems.00167-22</a></p><p><b>Distinct diet-microbiota-metabolism interactions in overweight and obese pregnant women: A metagenomics approach</b><br>Lotankar
M,
Mokkala
K,
Houttu
N,
Koivuniemi
E,
Sørensen
N,
Nielsen
H,
Munukka
E,
Lahti
L & 
Laitinen
K.<br>Microbiology Spectrum 10,
2022<br><a href=https://doi.org/10.1128/spectrum.00893-21>10.1128/spectrum.00893-21</a>
|
<a href=https://journals.asm.org/doi/abs/10.1128/spectrum.00893-21>URL</a></p><p><b>Combined effects of host genetics and diet on human gut microbiota and incident disease in a single population cohort</b><br>Qin
Y,
Havulinna
A,
Liu
Y,
Jousilahti
P,
Ritchie
S,
Tokolyi
A,
Sanders
J,
Valsta
L,
Brożyńska
M,
Zhu
Q,
Tripathi
A,
Vázquez-Baeza
Y,
Loomba
R,
Cheng
S,
Jain
M,
Niiranen
T,
Lahti
L,
Knight
R,
Salomaa
V,
Inouye
M & 
Méric
G.<br>Nature Genetics 54(2),
2022<br><a href=https://doi.org/10.1038/s41588-021-00991-z>10.1038/s41588-021-00991-z</a></p><p><b>Gut microbiome composition is predictive of incident type 2 diabetes in a population cohort of 5 572 finnish adults</b><br>Ruuskanen
M,
Erawijantari
P,
Havulinna
A,
Liu
Y,
Méric
G,
Tuomilehto
J,
Inouye
M,
Jousilahti
P,
Salomaa
V,
Jain
M,
Knight
R,
Lahti*
L & 
Niiranen*
T.<br>Diabetes Care 45(4),
2022<br><a href=https://doi.org/10.2337/dc21-2358>10.2337/dc21-2358</a></p><p><b>Efficient computation of Faith’s phylogenetic diversity with applications in characterizing microbiomes</b><br>Armstrong
G,
Cantrell
K,
Huang
S,
McDonald
D,
Haiminen
N,
Carrieri
A,
Zhu
Q,
Gonzalez
A,
McGrath
I,
Beck
K,
Hakim
D,
Havulinna
A,
Méric
G,
Niiranen
T,
Lahti
L,
Salomaa
V,
Jain
M,
Inouye
M,
Swafford
A,
Kim
H,
Parida
L,
Vázquez-Baeza
Y & 
Rob Knight
.<br>Genome Research 31,
2021<br><a href=https://doi.org/10.1101/gr.275777.121>10.1101/gr.275777.121</a></p><p><b>Impact of combined consumption of fish oil and probiotics on the serum metabolome in pregnant women with overweight or obesity</b><br>Mokkala
K,
Vahlberg
T,
Houttu
N,
Koivuniemi
E,
Lahti
L & 
Laitinen
K.<br>EbioMedicine 73,
2021<br><a href=https://doi.org/10.1016/j.ebiom.2021.103655>10.1016/j.ebiom.2021.103655</a></p><p><b>Xylo-oligosaccharides in prevention of hepatic steatosis and adipose tissue inflammation: Associating taxonomic and metabolomic patterns in fecal microbiomes with biclustering</b><br>Hintikka
J,
Lensu
S,
Mäkinen
E,
Karvinen
S,
Honkanen
M,
Linden
J,
Garrels
T,
Pekkala
S & 
Lahti
L.<br>International Journal of Environmental Research and Public Health 18(8),
2021<br><a href=https://doi.org/10.3390/ijerph18084049>10.3390/ijerph18084049</a></p><p><b>Gut microbiota diversity but not composition is related to saliva cortisol stress response at the age of 2.5 months</b><br>Keskitalo
A,
Aatsinki
A,
Kortesluoma
S,
Pelto
J,
Korhonen
L,
Lahti
L,
Lukkarinen
M,
Munukka
E,
Karlsson
H & 
Karlsson
L.<br>Stress 24(5),
2021<br>In press.<br><a href=https://doi.org/10.1080/10253890.2021.1895110>10.1080/10253890.2021.1895110</a></p><p><b>Microbiome data science in the <i>SummarizedExperiment</i> universe</b><br>Lahti
L,
Ernst
F,
Shetty
S & 
others.<br>F1000Research 10(748),
2021<br>(slides); version 1; not peer reviewed<br><a href=https://doi.org/10.7490/f1000research.1118712.1>10.7490/f1000research.1118712.1</a></p><p><b>Applications of machine learning in human microbiome studies: A review on feature selection, biomarker identification, disease prediction and treatment</b><br>Marcos-Zambrano
L,
Karaduzovic-Hadziabdic
K,
Loncar Turukalo
T,
Przymus
P,
Trajkovik
V,
Aasmets
O,
Berland
M,
Gruca
A,
Hasic
J,
Hron
K,
Klammsteiner
T,
Kolev
M,
Lahti
L,
Lopes
M,
Moreno
V,
Naskinova
I,
Org
E,
Paciência
I,
Papoutsoglou
G,
Shigdel
R,
Stres
B,
Vilne
B,
Yousef
M,
Zdravevski
E,
Tsamarindos
I,
Santa Pau
E,
Claesson
M,
Moreno-Indias
I & 
Truu
J.<br>Frontiers in Microbiology 12,
2021<br><a href=https://doi.org/10.3389/fmicb.2021.634511>10.3389/fmicb.2021.634511</a>
|
<a href=https://www.frontiersin.org/article/10.3389/fmicb.2021.634511>URL</a></p><p><b>Statistical and machine learning techniques in human microbiome studies: Contemporary challenges and solutions</b><br>Moreno-Indias
I,
Lahti
L,
Nedyalkova
M,
Elbere
I,
Roshchupkin
G,
Adilovic
M,
Aydemir
O,
Bakir-Gungor
B,
Pau
E,
D’Elia
D,
Desai
M,
Falquet
L,
Gundogdu
A,
Hron
K,
Klammsteiner
T,
Lopes
M,
Zambrano
L,
Marques
C,
Mason
M,
May
P,
Pašić
L,
Pio
G,
Pongor
S,
Promponas
V,
Przymus
P,
Sáez-Rodríguez
J,
Sampri
A,
Shigdel
R,
Stres
B,
Suharoschi
R,
Truu
J,
Truică
C,
Vilne
B,
Vlachakis
D,
Yılmaz
E,
Zeller
G,
Zomer
A,
Gómez-Cabrero
D & 
Claesson
M.<br>Frontiers in Microbiology 12,
2021<br><a href=https://doi.org/10.3389/fmicb.2021.635781>10.3389/fmicb.2021.635781</a></p><p><b>Targeting gut microbiota to treat hypertension: A systematic review</b><br>Palmu
J,
Lahti
L & 
Niiranen
T.<br>International Journal of Environmental Research and Public Health 18(3),
2021<br><a href=https://doi.org/10.3390/ijerph18031248>10.3390/ijerph18031248</a></p><p><b>Modeling spatial patterns in host-associated microbial communities</b><br>Ruuskanen
M,
Sommeria-Klein
G,
Havulinna
A,
Niiranen
T & 
Lahti
L.<br>Environmental Microbiology 23(5),
2021<br><a href=https://doi.org/10.1111/1462-2920.15462>10.1111/1462-2920.15462</a></p><p><b>Links between gut microbiome composition and fatty liver disease in a large population sample</b><br>Ruuskanen
M,
Åberg
F,
Männistö
V,
Havulinna
A,
Méric
G,
Liu
Y,
Loomba
R,
Vázquez-Baeza
Y,
Tripathi
A,
Valsta
L,
Inouye
M,
Jousilahti
P,
Salomaa
V,
Jain
M,
Knight
R,
Lahti
L & 
Niiranen
T.<br>Gut Microbes 13,
2021<br><a href=https://doi.org/10.1080/19490976.2021.1888673>10.1080/19490976.2021.1888673</a></p><p><b>Taxonomic signatures of cause-specific mortality risk in human gut microbiome</b><br>Salosensaari*
A,
Laitinen*
V,
Havulinna
A,
Meric
G,
Cheng
S,
Perola
M,
Valsta
L,
Alfthan
G,
Inouye
M,
Watrous
J,
Long
T,
Salido
R,
Sanders
K,
Brennan
C,
Humphrey
G,
Sanders
J,
Jain
M,
Jousilahti
P,
Salomaa
V,
Knight
R,
Lahti*
L & 
Niiranen*
T.<br>Nature Communications 12,
2021<br><a href=https://doi.org/10.1038/s41467-021-22962-y>10.1038/s41467-021-22962-y</a></p><p><b>Gut microbiota of patients with different subtypes of gastric cancer and gastrointestinal stromal tumors</b><br>Sarhadi
V,
Mathew
B,
A
A,
Karla
T,
Tikkanen
M,
Rautelin
H,
Lahti
L,
Puolakkainen
P & 
Knuutila
S.<br>Gut Pathogens 13,
2021<br><a href=https://doi.org/10.1186/s13099-021-00403-x>10.1186/s13099-021-00403-x</a></p><p><b>Systemic cross-talk between brain, gut, and peripheral tissues in glucose homeostasis: Effects of exercise training (CROSSYS): Exercise training intervention in monozygotic twins discordant for body weight</b><br>Heiskanen
M,
Honkala
S,
Ojala
R,
Lautamäki
R,
Koskensalo
K,
Lietzén
M,
Hentilä
J,
Saunavaara
V,
Saunavaara
J,
Helmiö
M,
Löyttyniemi
E,
Nummenmaa
L,
Collado
M,
Malm
T,
Lahti
L,
Pietiläinen
K,
Kaprio
J,
Rinne
J & 
Hannukainen
J.<br>BMC Sports Science, Medicine and Rehabilitation 13,
2021<br><a href=https://doi.org/10.1186/s13102-021-00241-z>10.1186/s13102-021-00241-z</a></p><p><b>Early prediction of liver disease using conventional risk factors and gut microbiome-augmented gradient boosting</b><br>Liu
Y,
Méric
G,
Havulinna
A,
Teo
S,
Åberg
F,
Ruuskanen
M,
Sanders
J,
Zhu
Q,
Tripathi
A,
Verspoor
K,
Cheng
S,
Jain
M,
Jousilahti
P,
Vázquez-Baeza
Y,
Loomba
R,
Lahti
L,
Niiranen
T,
Salomaa
V,
Knight
R & 
Inouye
M.<br>Cell Metabolism 34,
2022<br><a href=https://doi.org/10.1016/j.cmet.2022.03.002>10.1016/j.cmet.2022.03.002</a></p><p><b>Maternal prenatal psychological distress and hair cortisol levels associate with infant fecal microbiota composition at 2.5 months of age</b><br>Aatsinki
A,
Keskitalo
A,
Laitinen
V,
Munukka
E,
Uusitupa
H,
Lahti
L,
Kortesluoma
S,
Mustonen
P,
Rodrigues
A,
Coimbra
B,
Huovinen
P,
Karlsson
H & 
Karlsson
L.<br>Psychoneuroendocrinology 119,
2020<br><a href=https://doi.org/10.1016/j.psyneuen.2020.104754>10.1016/j.psyneuen.2020.104754</a></p><p><b>Partial restoration of normal intestinal microbiota in morbidly obese women six months after bariatric surgery</b><br>Koffert
J,
Lahti
L,
Nylund
L,
Salminen
S,
Hannukainen
J,
Salminen
P,
Vos
W & 
Nuutila
P.<br>PeerJ – Life & Environment 8,
2020<br>Shared first authorship.<br><a href=https://doi.org/10.7717/peerj.10442>10.7717/peerj.10442</a></p><p><b>Diet, perceived intestinal well-being, fecal microbiota and short chain fatty acids in oat-using subjects with celiac disease or gluten sensitivity.</b><br>Nylund
L,
Hakkola
S,
Lahti
L,
Salminen
S,
Kalliomäki
M,
Yang
B & 
Linderborg
K.<br>Nutrients 12,
2020<br>Special issue.<br><a href=https://doi.org/10.3390/nu12092570>10.3390/nu12092570</a></p><p><b>Association between the gut microbiota and blood pressure in a population cohort of 6953 individuals</b><br>Palmu
J,
Salosensaari
A,
Havulinna
A,
Cheng
S,
Inouye
M,
Jain
M,
Salido
R,
Sanders
K,
Brennan
C,
Humphrey
G,
Sanders
J,
Vartiainen
E,
Laatikainen
T,
Jousilahti
P,
Salomaa
V,
Knight
R,
Lahti
L & 
Niiranen
T.<br>Journal of the American Heart Association
2020<br><a href=https://doi.org/10.1161/JAHA.120.016641>10.1161/JAHA.120.016641</a></p><p><b>Eicosanoid inflammatory mediators are robustly associated with blood pressure in the general population</b><br>Palmu
J & 
others.<br>Journal of American Heart Association 9,
2020<br><a href=https://doi.org/10.1161/JAHA.120.017598>10.1161/JAHA.120.017598</a></p><p><b>Prebiotic xylo-oligosaccharides targeting faecalibacterium prausnitzii prevent high fat diet-induced hepatic steatosis in rats</b><br>Lensu
S,
Pariyani
R,
Mäkinen
E,
Yang
B,
Saleem
W,
Munukka
E,
Lehti
M,
Yang
B,
Linden
J,
Tiirola
M,
Lahti
L & 
Pekkala
S.<br>Nutrients 12(11),
2020<br><a href=https://doi.org/10.3390/nu12113225>10.3390/nu12113225</a></p><p><b>Gut microbiota and host gene mutations in colorectal cancer patients and controls of iranian and finnish origin</b><br>Sarhadi
V,
Lahti
L,
Saberi
F,
Youssef
O,
Kokkola
A,
Karla
T,
Tikkanen
M,
Rautelin
H,
Puolakkainen
P,
Salehi
R & 
Knuutila
S.<br>Anticancer Research 40(3),
2020<br><a href=https://doi.org/10.21873/anticanres.14074>10.21873/anticanres.14074</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2020-Sarhadi-Anticanc.pdf>PDF</a></p><p><b>Unsupervised hierarchical clustering identifies a metabolically challenged subgroup of hypertensive individuals</b><br>Vaura
F,
Salomaa
V,
Kantola
I,
Kaaja
R,
Lahti
L & 
Niiranen
T.<br>Journal of Clinical Hypertension 22(9),
2020<br><a href=https://doi.org/10.1111/jch.13984>10.1111/jch.13984</a></p><p><b>Gut microbiota composition is associated with temperament traits in infants</b><br>Aatsinki
A,
Lahti
L,
Uusitupa
H,
Munukka
E,
Anniina Keskitalo
S,
O’Mahony
S,
Pietilä
S,
Elo
L,
Eerola
E,
Karlsson
H & 
Karlsson
L.<br>Brain Behavior and Immunity 80,
2019<br><a href=https://doi.org/10.1016/j.bbi.2019.05.035>10.1016/j.bbi.2019.05.035</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2019-Aatsinki.pdf>PDF</a></p><p><b>Microbiome data science</b><br>Shetty
S & 
Lahti
L.<br>Journal of Biosciences 44,
2019<br><a href=https://doi.org/10.1007/s12038-019-9930-2>10.1007/s12038-019-9930-2</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2019-Shetty-MDS.pdf>PDF</a></p><p><b>Microbial communities in a dynamic it in vitro model for the human ileum resemble the human ileal microbiota</b><br>Stolaki
M,
Minekus
M,
Venema
K,
Lahti
L,
Smid
E,
Kleerebezem
M & 
Zoetendal
E.<br>FEMS Microbiology Ecology(8),
2019<br><a href=https://doi.org/10.1093/femsec/fiz096>10.1093/femsec/fiz096</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2019-Stolaki-FEMS.pdf>PDF</a></p><p><b>Gut microbiota composition in mid-pregnancy is associated with gestational weight gain but not prepregnancy body mass index</b><br>Aatsinki
A,
Uusitupa
H,
Munukka
E,
Pesonen
H,
Rintala
A,
Pietilä
S,
Lahti
L,
Eerola
E,
Karlsson
L & 
Karlsson
H.<br>Journal of Womens Health
2018<br>Epub ahead of print.<br><a href=https://doi.org/10.1089/jwh.2017.6488>10.1089/jwh.2017.6488</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2018-Aatsinki.pdf>PDF</a></p><p><b>Signatures of ecological processes in microbial community time series</b><br>Faust
K,
Bauchinger
F,
Laroche
B,
Buyl
S,
Lahti
L,
Washburne
A,
Gonze
D & 
Widder
S.<br>Microbiome 6(120),
2018<br><a href=https://doi.org/10.1186/s40168-018-0496-2>10.1186/s40168-018-0496-2</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2018-Faust-Microbiome.pdf>PDF</a></p><p><b>Microbial communities as dynamical systems</b><br>Gonze
D,
Coyte
K,
Lahti
L & 
Faust
K.<br>Current Opinion in Microbiology 44,
2018<br><a href=https://doi.org/10.1016/j.mib.2018.07.004>10.1016/j.mib.2018.07.004</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2018-Gonze.pdf>PDF</a></p><p><b>Stool microbiota composition differs in patients with stomach, colon, and rectal neoplasms</b><br>Youssef
O,
Lahti
L,
Kokkola
A,
Karla
T,
Tikkanen
M,
Ehsan
H,
Carpelan‑Holmström
M,
Koskensalo
S,
Böhling
T,
Rautelin
H,
Puolakkainen
P,
Knuutila
S & 
Sarhadi
V.<br>Digestive Diseases and Sciences 63(11),
2018<br><a href=https://doi.org/10.1007/s10620-018-5190-5>10.1007/s10620-018-5190-5</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2018-Youssef.pdf>PDF</a></p><p><b>Comparative gut microbiota and resistome profiling of intensive care patients receiving selective digestive tract decontamination and healthy subjects</b><br>Buelow
E,
González
T,
Fuentes
S,
Steenhuijsen Piters
W,
Lahti
L,
Bayjanov
J,
Majoor
E,
Braat
J,
Mourik
M,
Oostdijk
E,
Willems
R,
Bonten
M,
Passel
M,
Smidt
H & 
Schaik
W.<br>Microbiome 5(1),
2017<br><a href=https://doi.org/10.1186/s40168-017-0309-z>10.1186/s40168-017-0309-z</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2017-Buelow-Microbiome.pdf>PDF</a></p><p><b>Multi-stability and the origin of microbial community types</b><br>Gonze
D,
Lahti
L,
Raes
J & 
Faust
K.<br>ISME Journal 11,
2017<br><a href=https://doi.org/10.1038/ismej.2017.60>10.1038/ismej.2017.60</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2017-Faust-ISME.pdf>PDF</a></p><p><b>Linking statistical and ecological theory: Hubbell’s unified neutral theory of biodiversity as a hierarchical dirichlet process</b><br>Harris
K,
Parsons
T,
Ijaz
U,
Lahti
L,
Holmes
I & 
Quince
C.<br>Proceedings of the IEEE 105(3),
2017<br><a href=https://doi.org/10.1109/JPROC.2015.2428213>10.1109/JPROC.2015.2428213</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2017-Harris-IEEE.pdf>PDF</a></p><p><b>Intestinal microbiome landscaping: Insight in community assemblage and implications for microbial modulation strategies</b><br>Shetty
S,
Hugenholtz
F,
Lahti
L,
Smidt
H,
Vos
W & 
Danchin
A.<br>FEMS Microbiology Reviews 41,
2017<br>review<br><a href=https://doi.org/10.1093/femsre/fuw045>10.1093/femsre/fuw045</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2017-Shetty-FEMS.pdf>PDF</a></p><p><b>Metagenomics meets time series analysis: Unraveling microbial community dynamics</b><br>Faust
K,
Lahti
L,
Gonze
D,
Vos
W & 
Raes
J.<br>Current Opinion in Microbiology 25,
2015<br><a href=https://doi.org/10.1016/j.mib.2015.04.004>10.1016/j.mib.2015.04.004</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2015-Faust-CurrOpMicrob.pdf>PDF</a></p><p><b>Fat, fiber and cancer risk in african, americans and rural africans</b><br>O’Keefe
S,
Li
J,
Lahti
L,
Ou
J,
Carbonero
F,
Mohammed
K,
Posma
J,
Kinross
J,
Wahl
E,
Ruder
E,
Vipperla
K,
Naidoo
V,
Mtshali
L,
Tims
S,
PGB Puylaert
J,
Krasinskas
A,
Benefiel
A,
Kaseb
H,
Newton
K,
Nicholson
J,
Vos
W,
Gaskins
H & 
Zoetendal
E.<br>Nature Communications 6,
2015<br>Top science article in Guardian on the release day.<br><a href=https://doi.org/10.1038/ncomms7342>10.1038/ncomms7342</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2015-OKeefe-NComms.pdf>PDF</a></p><p><b>Improved taxonomic assignment of human intestinal 16S rRNA sequences by a dedicated reference database</b><br>Ritari
J,
Salojärvi
J,
Lahti
L & 
Vos
W.<br>BMC Genomics 16,
2015<br><a href=https://doi.org/10.1186/s12864-015-2265-y>10.1186/s12864-015-2265-y</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2015-Ritari-BMCGenomics.pdf>PDF</a></p><p><b>Impact of a wastewater treatment plant on microbial community composition and function in a hyporheic zone of a eutrophic river</b><br>Atashgahi
S,
Aydin
R,
Dimitrov
M,
Sipkema
D,
Hamonts
K,
Lahti
L,
Maphosa
F,
Kruse
T,
Saccenti
E,
Dirk Springael
W & 
Smidt
H.<br>Scientific Reports 5(17284),
2015<br><a href=https://doi.org/10.1038/srep17284>10.1038/srep17284</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2015-Atagashi.pdf>PDF</a></p><p><b>A novel atlas of gene expression in human skeletal muscle reveals molecular changes associated with aging</b><br>Su
J,
Ekman
C,
Oskolkov
N,
Lahti
L,
Ström
K,
Brazma
A,
Groop
L,
Rung
J & 
Ola Hansson
.<br>Skeletal Muscle 5(35),
2015<br><a href=https://doi.org/10.1186/s13395-015-0059-1>10.1186/s13395-015-0059-1</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2015-Su-SkeletalMuscle.pdf>PDF</a></p><p><b>Binning metagenomic contigs by coverage and composition</b><br>Alneberg
J,
Bjarnason
B,
Bruijn
I,
Schirmer
M,
Quick
J,
Ijaz
U,
Lahti
L,
Loman
N,
Andersson
A & 
Quince
C.<br>Nature Methods 11(1144–1146),
2014<br>CONCOCT algorithm<br><a href=https://doi.org/10.1038/nmeth.3103>10.1038/nmeth.3103</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2014-Alneberg-NatMeth.pdf>PDF</a></p><p><b>Systematic use of computational methods allows stratifying treatment responders in glioblastoma multiforme</b><br>Louhimo
R,
Aittomäki
V,
Faisal
A,
Laakso
M,
Chen
P,
Ovaska
K,
Valo
E,
Lahti
L,
Rogojin
V,
Kaski
S & 
Hautaniemi.
S.<br>Systems Biomedicine (special issue) 1,
2014<br>Critical Assessment of Massive Data Analysis (CAMDA) workshop<br><a href=https://doi.org/10.4161/sysb.28904>10.4161/sysb.28904</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2014-Louhimo.pdf>PDF</a></p><p><b>Tipping elements in the human intestinal ecosystem</b><br>Lahti
L,
Salojärvi
J,
Salonen
A,
Scheffer
M & 
Vos
W.<br>Nature Communications 5,
2014<br><a href=https://doi.org/10.1038/ncomms5344>10.1038/ncomms5344</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2014-Lahti-NatComm.pdf>PDF</a></p><p><b>Impact of diet and individual variation on intestinal microbiota composition and fermentation products in obese men</b><br>Salonen
A,
Lahti
L,
Salojärvi
J,
Holtrop
G,
Korpela
K,
Duncan
S,
Date
P,
Johnstone
A,
Lobley
G,
Louis
P,
Flint
H & 
Vos
W.<br>ISME Journal 8,
2014<br><a href=https://doi.org/10.1038/ismej.2014.63>10.1038/ismej.2014.63</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2014-Salonen-ISME.pdf>PDF</a></p><p><b>Copy number alterations and neoplasia specific mutations in MELK, PDCD1LG2, TLN1, and PAX5 at 9p in different neoplasias</b><br>Sarhadi
V,
Lahti
L,
Scheinin
I,
Ellonen
P,
Kettunen
E,
Serra
M,
Scotlandi
K,
Picci
P & 
Knuutila
S.<br>Genes, Chromosomes and Cancer 53(7),
2014<br><a href=https://doi.org/10.1002/gcc.22168>10.1002/gcc.22168</a></p><p><b>A fully scalable online pre-processing algorithm for short oligonucleotide microarray atlases</b><br>Lahti
L,
Torrente
A,
Elo
L,
Brazma
A & 
Rung
J.<br>Nucleic Acids Research 41(10),
2013<br>Implementation available through Bioconductor RPA package.<br><a href=https://doi.org/10.1093/nar/gkt229>10.1093/nar/gkt229</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2013-Lahti-NAR.pdf>PDF</a></p><p><b>Associations between the human intestinal microbiota, lactobacillus rhamnosus GG and serum lipids indicated by integrated analysis of high-throughput profiling data</b><br>Lahti
L,
Salonen
A,
Kekkonen
R,
Salojärvi
J,
Jalanka-Tuovinen
J,
Palva
A,
Orešič
M & 
Vos
W.<br>PeerJ 1,
2013<br>Highly accessed, featured in PeerJ journal blog and Top-10 Microbiology collection.<br><a href=https://doi.org/10.7717/peerj.32>10.7717/peerj.32</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2013-Lahti-PeerJ.pdf>PDF</a></p><p><b>Cancer gene prioritization by integrative analysis of mRNA expression and DNA copy number data: A comparative review</b><br>Lahti
L,
Schäfer
M,
Klein
H,
Bicciato
S & 
Dugas
M.<br>Briefings in Bioinformatics 14(1),
2013<br><a href=https://doi.org/10.1093/bib/bbs005>10.1093/bib/bbs005</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2013-Lahti-BriefBioinf.pdf>PDF</a></p><p><b>Targeted resequencing of 9p in acute lymphoblastic leukemia yields concordant results with array CGH and reveals novel genomic alterations</b><br>Sarhadi
V,
Lahti
L,
Scheinin
I,
Tyybäkinoja
A,
Savola
S,
Usvasalo
A,
Räty
R,
Elonen
E,
Ellonen
P,
Saarinen-Pihkala
U & 
Knuutila
S.<br>Genomics 102(3),
2013<br><a href=https://doi.org/10.1016/j.ygeno.2013.01.001>10.1016/j.ygeno.2013.01.001</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2013-Sarhadi-Genomics.pdf>PDF</a></p><p><b>Intestinal microbiota, individuality and health. In: Computational methods aiding early-stage drug design (dagstuhl seminar 13212)</b><br>Lahti
L<br>Dagstuhl Reports 3(5) Schloss Dagstuhl–Leibniz-Zentrum fuer Informatik,
2013<br><a href=https://doi.org/10.4230/DagRep.3.5.78>10.4230/DagRep.3.5.78</a>
|
<a href=http://drops.dagstuhl.de/opus/volltexte/2013/4179>URL</a></p><p><b>The adult intestinal core microbiota is determined by analysis depth and health status</b><br>Salonen
A,
Salojärvi
J,
Lahti
L & 
Vos
.<br>Clinical Microbiology and Infection 18(Suppl. 4),
2012<br><a href=https://doi.org/10.1111/j.1469-0691.2012.03855.x>10.1111/j.1469-0691.2012.03855.x</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2012-Salonen-CMI.pdf>PDF</a></p><p><b>MicroRNA profiling predicts survival in anti-EGFR treated chemorefractory metastatic colorectal cancer patients with wild-type KRAS and BRAF</b><br>Mosakhani
N,
Lahti
L,
Borze
I,
Karjalainen-Lindsberg
M,
Sundstrom
J,
Ristamaki
R,
Osterlund
P,
Knuutila
S & 
Virinder Kaur Sarhadi
.<br>Cancer Genetics 205(11),
2012<br><a href=https://doi.org/10.1016/j.cancergen.2012.08.003>10.1016/j.cancergen.2012.08.003</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2012-Mosakhani-CG.pdf>PDF</a></p><p><b>MicroRNA profiling in pediatric acute lymphoblastic leukemia: Novel prognostic tools</b><br>Mosakhani
N,
Sarhadi
V,
Usvasalo
A,
Karjalainen-Lindsberg
M,
Lahti
L,
Tuononen
K,
Saarinen-Pihkala
U & 
Knuutila
S.<br>Leukemia & Lymphoma 53(12),
2012<br><a href=https://doi.org/10.3109/10428194.2012.685731>10.3109/10428194.2012.685731</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2012-Mosakhani-LL.pdf>PDF</a></p><p><b>Homozygous deletions of cadherin genes in chondrosarcoma – an array CGH study</b><br>Niini
T,
Scheinin
I,
Lahti
L,
Savola
S,
Mertens
F,
Hollmén
J,
Böhling
T,
Kivioja
A,
Nord
K & 
Knuutila
S.<br>Cancer Genetics 205(11),
2012<br><a href=https://doi.org/10.1016/j.cancergen.2012.09.007>10.1016/j.cancergen.2012.09.007</a></p><p><b>MicroRNA microarrays on archive bone marrow core biopsies of leukemias - method validation</b><br>Borze
I,
Guled
M,
Musse
S,
Raunio
A,
Elonen
E,
Saarinen-Pihkala
U,
Karjalainen-Lindsberg
M,
Lahti
L & 
Knuutila.
S.<br>Leukemia Research 35,
2011<br><a href=https://doi.org/10.1016/j.leukres.2010.08.005>10.1016/j.leukres.2010.08.005</a></p><p><b>Biomarker discovery via dependency analysis of multi-view functional genomics data</b><br>Faisal
A,
Louhimo
R,
Lahti
L,
Hautaniemi
S & 
Kaski
S.<br>NIPS 2011 workshop from statistical genetics to predictive models in personalized medicine
2011<br><a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2011-Faisal-NIPS.pdf>PDF</a></p><p><b>Intestinal microbiota in healthy adults: Temporal analysis reveals individual and common core and relation to intestinal symptoms</b><br>Jalanka-Tuovinen
J,
Salonen
A,
Nikkilä
J,
Immonen
O,
Kekkonen
R,
Lahti
L,
Palva
A & 
Vos
W.<br>PLoS One 6(7),
2011<br><a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2011-JalankaTuovinen-PLoS.pdf>PDF</a></p><p><b>Intcomp: A benchmarking pipeline for integrative cancer gene detection algorithms</b><br>Lahti
L,
Schäfer
M,
Klein
H,
Bicciato
S & 
Dugas
M.<br>software,
2011<br>R-forge<br><a href=http://intcomp.r-forge.r-project.org/>URL</a></p><p><b>Probabilistic dependency models for data integration in functional genomics</b><br>Lahti
L & 
Kaski
S.<br>2011
Machine Learning for Systems Biology workshop, ISMB, Vienna, Austria<br><a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2011-MLSB-Lahti.pdf>PDF</a></p><p><b>Probabilistic analysis of probe reliability in differential gene expression studies with short oligonucleotide arrays</b><br>Lahti
L,
Elo
L,
Aittokallio
T & 
Kaski
S.<br>IEEE/ACM Transactions on Computational Biology and Bioinformatics 8(1) IEEE Computer Society,
2011<br><a href=https://doi.org/10.1109/TCBB.2009.38>10.1109/TCBB.2009.38</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2011-Lahti-TCBB.pdf>PDF</a></p><p><b>Array comparative genomic hybridization reveals frequent alterations of G1/s checkpoint genes in undifferentiated pleomorphic sarcoma of bone</b><br>Niini
T,
Lahti
L,
Michelacci
F,
Ninomiya
S,
Hattinger
C,
Guled
M,
Böhling
T,
Picci
P,
Serra
M & 
Knuutila
S.<br>Genes, Chromosomes and Cancer 50(5),
2011<br><a href=https://doi.org/10.1002/gcc.20851>10.1002/gcc.20851</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2011-Niini-GCC.pdf>PDF</a></p><p><b>Integrative analysis of microRNA, mRNA and aCGH data reveals asbestos- and histology-related changes in lung cancer</b><br>Nymark
P,
Guled
M,
Borze
I,
Faisal
A,
Lahti
L,
Salmenkivi
K,
Kettunen
E,
Anttila
S & 
Knuutila
S.<br>Genes, Chromosomes and Cancer 50(8),
2011<br><a href=https://doi.org/10.1002/gcc.20880>10.1002/gcc.20880</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2011-Nymark-GCC.pdf>PDF</a></p><p><b>Global modeling of transcriptional responses in interaction networks</b><br>Lahti
L,
Knuuttila
J & 
Kaski
S.<br>Bioinformatics 26,
2010<br>R/Matlab implementations available at http://netpro.r-forge.r-project.org/<br><a href=https://doi.org/10.1093/bioinformatics/btq500>10.1093/bioinformatics/btq500</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2010-Bioinformatics-Lahti-NetResponse.pdf>PDF</a></p><p><b>Unique microRNA profile in dupuytren’s contracture supports deregulation of beta-catenin pathway</b><br>Mosakhani
N,
Guled
M,
Lahti
L,
Borze
I,
Forsman
M,
Ryhänen
J & 
Knuutila
S.<br>Modern Pathology 23,
2010<br>International Dupuytren Award for best research paper in 2011<br><a href=https://doi.org/10.1038/modpathol.2010.146>10.1038/modpathol.2010.146</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2010-ModernPathology-Mosakhani-Dupuytre.pdf>PDF</a></p><p><b>CDKN2A, NF2 and JUN are dysregulated among other genes by miRNAs in malignant mesothelioma - a miRNA microarray analysis</b><br>Guled
M,
Lahti
L,
Lindholm
P,
Salmenkivi
K,
Bagwan
I,
Nicholson
A & 
Knuutila
S.<br>Genes, Chromosomes and Cancer 48(7),
2009<br><a href=https://doi.org/10.1002/gcc.20669>10.1002/gcc.20669</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2009-GCC-Guled-miRNA.pdf>PDF</a></p><p><b>RPA: Probe reliability and differential gene expression analysis</b><br>Lahti
L<br>software,
2009<br>R/Bioconductor: RPA<br><a href=https://doi.org/10.18129/B9.BIOC.RPA>10.18129/B9.BIOC.RPA</a>
|
<a href=http://bioconductor.org/packages/release/bioc/html/RPA.html>URL</a></p><p><b>A brief overview on the BioPAX and SBML standards</b><br>Lahti
L<br>arXiv,
2007<br>(in Finnish)<br><a href=https://arxiv.org/abs/1109.4919>URL</a></p><p><b>Gene expression profiles in asbestos-exposed epithelial and mesothelial lung cell lines</b><br>Nymark
P,
Lindholm
P,
Korpela
M,
Lahti
L,
Ruosaari
S,
Kaski
S,
Hollmen
J,
Anttila
S,
Kinnula
V & 
Knuutila
S.<br>BMC Genomics 8(62),
2007<br><a href=https://doi.org/10.1186/1471-2164-8-62>10.1186/1471-2164-8-62</a></p><p><b>Integrating probe-level expression changes across generations of affymetrix arrays</b><br>Elo
L,
Lahti
L,
Skottman
H,
Kyläniemi
M,
Lahesmaa
R & 
Aittokallio
T.<br>Nucleic Acids Research 33(22),
2005<br><a href=https://doi.org/10.1093/nar/gni193>10.1093/nar/gni193</a>
|
<a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2005-Elo-NAR.pdf>PDF</a></p><p><b>Associative clustering by maximizing a Bayes factor</b><br>Sinkkonen
J,
Nikkilä
J,
Lahti
L & 
Kaski
S.<br>Helsinki University of Technology and Laboratory of Computer and Information Science,
2003<br><a href=https://github.com/openresearchlabs/openresearchlabs.github.io/blob/master/content/publication_resources/papers/2003-TechRep-Sinkkonen-ACBayesFactor.pdf>PDF</a></p></div></div><div class=col-md-3></div></div></div></div><footer id=footer><div class=container><div class="col-md-4 col-sm-6"><h4>About us</h4><p>Turku Data Science Group.</p><hr class="hidden-md hidden-lg hidden-sm"></div><div class="col-md-4 col-sm-6"></div><div class="col-md-4 col-sm-6"><h4>Contact</h4><div class=social><script type=text/javascript>console.log("https://github.com/openresearchlabs/")</script><a href=https://github.com/openresearchlabs/ target=_blank style=opacity:1><i class='fab fa-2x fa-github'></i></a>
<script type=text/javascript>console.log("https://twitter.com/openreslabs")</script><a href=https://twitter.com/openreslabs target=_blank style=opacity:1><i class='fab fa-2x fa-twitter'></i></a>
<script type=text/javascript>console.log("mailto:leo.lahti@utu.fi")</script><a href=mailto:leo.lahti@utu.fi target=_blank style=opacity:1><i class='fas fa-2x fa-envelope'></i></a></div><p class=text-uppercase><strong>Turku Data Science Group</strong><br>University of Turku<br>Department of Computing<br>Turku<br>Finland<br><strong></strong></p><a href=/contact class="btn btn-small btn-template-main">Go to contact page</a><hr class="hidden-md hidden-lg hidden-sm"></div></div></footer><div id=copyright><div class=container><div class=col-md-12><p class=pull-left>Copyright (c) 2020-2024, Leo Lahti; some rights reserved.</p><p class=pull-right>Template by <a href=https://bootstrapious.com/p/universal-business-e-commerce-template>Bootstrapious</a>.
Ported to Hugo by <a href=https://github.com/devcows/hugo-universal-theme>DevCows</a>.</p></div></div></div></div><script async src="https://www.googletagmanager.com/gtag/js?id=G-3HREPRC83G"></script><script>var dnt,doNotTrack=!1;if(!1&&(dnt=navigator.doNotTrack||window.doNotTrack||navigator.msDoNotTrack,doNotTrack=dnt=="1"||dnt=="yes"),!doNotTrack){window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments)}gtag("js",new Date),gtag("config","G-3HREPRC83G")}</script><script src=//code.jquery.com/jquery-3.1.1.min.js integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin=anonymous></script><script src=//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js integrity=sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa crossorigin=anonymous></script><script src=//cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js></script><script src=//cdnjs.cloudflare.com/ajax/libs/waypoints/4.0.1/jquery.waypoints.min.js></script><script src=//cdnjs.cloudflare.com/ajax/libs/Counter-Up/1.0/jquery.counterup.min.js></script><script src=//cdnjs.cloudflare.com/ajax/libs/jquery-parallax/1.1.3/jquery-parallax.js></script><script src="//maps.googleapis.com/maps/api/js?v=3.exp"></script><script src=/js/hpneo.gmaps.js></script><script src=/js/gmaps.init.js></script><script src=/js/front.js></script><script src=/js/owl.carousel.min.js></script></body></html>